pupil-spa/src/app/accessibility-statement/accessibility-statement.component.html

```diff
@@ -4,12 +4,15 @@
   <app-breadcrumbs [breadcrumbs]="[{ url: '.', name: 'Home' }, { name: 'Accessibility Statement' } ]"></app-breadcrumbs>
   <div class="grid-row">
     <div class="column-two-thirds">
-      <h1 class="heading-xlarge">Accessibility statement for the multiplication tables check (MTC) service</h1>
+      <h1 class="heading-xlarge">Accessibility statement for the multiplication tables check service</h1>
       <p>
-        This statement applies to content published on the
-        <a class="govuk-link" href="https://pupil.multiplication-tables-check.service.gov.uk/">MTC pupil check application</a> and school administration area of the MTC.
-        We're constantly working to make the MTC accessible and user-friendly for all pupils, regardless of any physical,
-        cognitive or emotional impairment.
+        This statement applies to content published on the multiplication tables check (MTC) service.
+        This includes the school administration application accessed via DfE Sign-in and the pupil try it out and official
+        <a class="govuk-link" href="https://pupil.multiplication-tables-check.service.gov.uk/">check application.</a>
```

Comment:
This link is from the SPA to the SPA but will only work in production.  Why not use a relative path that will work for all environments?

Reply:
change to /